Constants.h

```diff
@@ -1,4 +1,4 @@
 #pragma once
 
-static const unsigned int numCachedStats = 52;
+static const unsigned int numCachedStats = 53;
```

Comment:
@mathchamp93 this doesn't affect saving does it?

Reply:
I don't believe the stat cache gets saved at all (that might change in the future though depending on what approach I use to deal with potential issues related to stats being checked in the middle of the game loading).

Even if it did, changing the number of stats would be automatically accommodated (it would break saved runs from old versions but the actual saving code would have been coded to use numCachedStats).

What the cache is used for is that when the game loads the max health will be cached immediately so that the cached value is used for the remainder of the loading process. Hyperspace directly saves and loads the health and max health of crew.

Also in the long run I might even remove the cache; it was an optimization I added before I found the root cause of performance issues (the custom augment code was poorly optimized and some augments got checked per crewmember so ships with lots of crew and/or augments would eat up a lot of CPU). I'd test the performance impact first, though.